docker-compose-services/redis/README.md

```diff
@@ -1,17 +1,93 @@
 # Redis
 
-This recipe adds a Redis container to a project.
+This recipe adds a Redis container to a project. See the [offical `redis` image](https://hub.docker.com/_/redis) page for supported versions and help.
 
 ## Installation
 
-* Copy the `commands/redis` directory to the commands folder of your project DDEV folder; you should end up with `.ddev/commands/redis/redis-cli` (make sure it's executable, `chmod +x .ddev/commands/redis/redis-cli`.)
-* If you need *Redis 5* change the image version as `redis:5` in `.ddev/docker-compose.redis.yaml`.
-* Update the example redis configuration file to your needs: `.ddev/redis/redis.conf`.
-* Start (or restart) DDEV to have the service initialized: `ddev start`
-* Your redis instance is available **inside the containers** at `ddev-<projectname>-redis:6379`:
-    * Host `ddev-${DDEV_SITENAME}-redis`
-    * Port `6379`
+* Copy the files in this folder (you can skip `readme.md`) into your project's `.ddev` folder. Your project's `.ddev` should contain these additional files.
 
-Now you can use `ddev redis-cli` to access the redis cli in the redis container.
+```
+.ddev/
+┣ commands/
+┃ ┣ redis/
+┃ ┃ ┗ redis-cli
+┣ redis/
+┃ ┗ redis.conf
+┗ docker-compose.redis.yaml
+```
 
+* Make sure the `redis-cli` command is executable
+
+```shell
+chmod +x .ddev/commands/redis/redis-cli
+```
+
+* To change the redis version, update the `image` line in `.ddev/docker-compose.redis.yaml`. EG. To use redis version `6`,
+
+```yaml
+# docker-compose.redis.yaml
+services:
+  redis:
+    image: redis:6
+```
+
+* To configure the redis service config file as needed: `.ddev/redis/redis.conf`
```

Comment:
[-]To[/-]

Reply:
Thanks ... I added it in my local version, but then remember you had the draft PR. Nice catch!